library/modules/bigip_device_info.py

```diff
@@ -6088,6 +6090,47 @@
       type: bool
       sample: yes
   sample: hash/dictionary of values
+users:
```

Comment:
maybe also add there what shell the user can access, to be in line with what the bigip_user returns too:

https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_user.py#L200

Reply:
Added. I was not sure how the shell works. Now, I'm using it as part of the checks!